es-vue-base/src/lib-components/EsProgress.vue

```diff
@@ -1,11 +1,14 @@
 <template>
-    <b-progress
-        class="EsProgress bg-gray-300 rounded-0"
-        :class="{ 'EsProgress--full': value === 100 }"
-        :height="height"
-        :value="value"
-        v-bind="$attrs"
-        v-on="$listeners" />
+    <div>
```

Comment:
Why are we wrapping this in a `div` now? This will result in some breaking changes. Could we wrap it where it's being used instead?